activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessExtensionsTest.java

```diff
@@ -150,4 +151,36 @@ public void processInstanceFailsIfVariableTypeIncorrect() {
                     .build());
         }).withMessage("Variables fail type validation: subscribe, name, age");
     }
+    
+    @Test
+    public void should_mapProcessVariables_when_startEventMappingExists() {
+
+        securityUtil.logInAs("user");
+
+        ProcessInstance process = processRuntime.start(ProcessPayloadBuilder.start()
+                .withProcessDefinitionKey("process-b42a166d-605b-4eec-8b96-82b1253666bf")
+                .withVariable("Text0xfems",
+                        "name_value")
+                .withVariable("Text0rvs0o",
+                        "email_value")
+                .withBusinessKey("my business key")
+                .build());
+
+        assertThat(process).isNotNull();
+        assertThat(process.getStatus()).isEqualTo(ProcessInstance.ProcessInstanceStatus.RUNNING);
+
+        List<VariableInstance> variableInstances = processRuntime.variables(ProcessPayloadBuilder.variables().withProcessInstance(process).build());
+
+        assertThat(variableInstances)
+        .isNotNull()
+        .hasSize(2)
+        .extracting(VariableInstance::getName,
+                    VariableInstance::getValue)
+        .containsOnly(
+                      tuple("name", "name_value"),
+                      tuple("email", "email_value")
+        );
+
+        processRuntime.delete(ProcessPayloadBuilder.delete(process));
```

Comment:
You can skip the process clean up here as the processes are already being deleted in `@After` method.

Reply:
Done